### DISCUSSION TITLE
There is an empty "-then function(){}" in every step

### DISCUSSION THREAD

**Discussion by amilcarjustino** [badeball/cypress-cucumber-preprocessor#1214] [General]
Hello.
In my projects, there is an empty "-then function(){}" in every step. 
Does anyone know why? Is there a way to omit this log?
Thanks in advance

<img width="272" alt="image" src="https://github.com/user-attachments/assets/59cbfd0f-c1d9-4245-b33c-a73f3a07e43a">

**Comment by badeball** [comment]
> Is there a way to omit this log?

This may be a bug. If you can provide a [reproducible example](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CONTRIBUTING.md#reduced-test-case), I can look into it.

**Reply by amilcarjustino** [reply]
Ok, thanks.
Created in https://github.com/amilcarjustino/example-cypress-cucumber-pre
Just need to open test duckduckgo.feature

**Reply by badeball** [reply]
Hi @amilcarjustino, I've fixed this with v20.1.1. Please check it out.

**Reply by amilcarjustino** [reply]
Hi @badeball 
I have tested it, and it is perfect.
Thank you very much for the support...
